docs/how-to/hip_runtime_api/opengl_interop.rst

```diff
@@ -0,0 +1,66 @@
+.. meta::
+   :description: HIP provides an OpenGL interoperability API that allows
+                 efficient data sharing between HIP's computing power and
+                 OpenGL's graphics rendering.
+   :keywords: AMD, ROCm, HIP, OpenGL, interop, interoperability
+
+********************************************************************************
+OpenGL interoperability
+********************************************************************************
+
+The HIP--OpenGL interoperation involves mapping OpenGL resources, such as
+buffers and textures, for HIP access. This mapping process enables HIP to
+utilize these resources directly, bypassing the need for data transfers between
+the CPU and GPU. This capability is useful in applications that require both
+intensive GPU computation and real-time visualization.
+
+The OpenGL resource can be mapped to HIP with :cpp:func:`hipGraphicsMapResources`
+function and the device pointers are accessed with
+:cpp:func:`hipGraphicsResourceGetMappedPointer` or
+:cpp:func:`hipGraphicsSubResourceGetMappedArray` functions.
+
+Unmapping resources with :cpp:func:`hipGraphicsUnmapResources` after
+computations ensure proper resource management.
+
+Example
+================================================================================
+
+ROCm examples have a `HIP--OpenGL interoperation example <https://github.com/ROCm/rocm-examples/tree/develop/HIP-Basic/opengl_interop>`_,
+where a simple HIP kernel is used to simulate a sine wave and rendered to a
+window as a grid of triangles using OpenGL. For a working example, there are
+multiple initialization steps needed like creating and opening a window, 
+initializing OpenGL or selecting the OpenGL-capable device. After the initialization
+in the example, the kernel simulates the sinewave and updates the window's
+framebuffer in a cycle until the window is not closed.
+
+.. note::
+
+   The more recent recent OpenGL functions loaded with `OpenGL loader <https://github.com/ROCm/rocm-examples/tree/develop/External/glad>`_,
```

Comment:
This sentence seems to be missing something, like "the more recent functions _are_ loaded"?

Reply:
Updated, please check.